restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/Conference.java

```diff
@@ -309,8 +326,10 @@ public void execute(Object message) throws Exception {
                 final Leave leave = new Leave();
                 call.tell(leave, super.source);
             }
+            //tell conference api client to call conference api to terminate conference and kick all calls
+            conferenceApiClient = conferenceApiClient();
```

Comment:
@maria-farooq Is there a way to tell if a conference has remote participants or not? This would be a great improvement in order to save resources by not creating and executing the ConferenceApiClient

Reply:
we can check by DB but that is like alternative to me not an optimisation, in most cases it would be an overhead query